First/ViewController.swift

```diff
@@ -76,33 +48,13 @@ class ViewController: UIViewController {
         password.text = "Password"
```

Comment:
Podobnie jak numerki, to wszystkie stringi też trzymamy w oddzielnym miejscu. Zazwyczaj na stringi robimy oddzielny plik, ale ty możesz póki co dać je do oddzielnych zmiennych. W tym przypadku to było by:
_private let passwordPlaceholder = "Password"_ lub coś w tym rodzaju

Reply:
☝️